[CU-gb1hy1] Add locales to the rest of files

Buglik opened the pull request:
Changed locales structure and added missing ones.

Buglik committed:
Reconstructing locales
todo (maybe): split tournament.ts file

Zummek commented:
Task linked: [CU-gb1hy1 Add locales to the rest of files](https://app.clickup.com/t/gb1hy1)

Buglik committed:
minor changes
locales status field changed
locales added type of tournament
teamRepository fixed minor error

Buglik committed:
misspelling fixed

Buglik committed:
prettier bug fixed
double quotes to single quotes

Zummek commented on web-app/src/components/tournament/GridListMobile.vue:
```diff
@@ -54,7 +54,7 @@
                 style="text-align:right"
                 :style="
-                  props.row.status === $t('tournament.isFinishedInProgress')
+                  props.row.status === $t('tournament.status.isFinished.inProgress')
```
it's old, update it

Zummek commented on web-app/src/components/tournament/creator/teamBuilder.vue:
```diff
@@ -199,10 +201,10 @@ export default class TeamBuilder extends Vue {
-      this.teamNameErrorMessage = this.$t('tournament.tooShortTeamNameError');
+      this.teamNameErrorMessage = this.$t('tournament.team.error.tooShortName');
     } else if (this.teamName.length > 40) {
       this.isErrorInputTeamName = true;
-      this.teamNameErrorMessage = this.$t('tournament.tooLongTeamNameError');
+      this.teamNameErrorMessage = this.$t('tournament.team.error.tooLongName');
     } else {
```
this code is required? When we have `maxlength` in component this will never happen. Right? i'm not sure. So this lines and `tooLongName` in locales are useless

Buglik replied:
true, it's useless after adding maxlength in input

Zummek reviewed (approved):
Look to comments, anyway LGTM

Buglik committed:
cleanup
removing unnecessary code
